### PR TITLE
Check if an option name is too long.

### PR DESCRIPTION
**Pull request by gmaurel**
This make possible to detect at the first run.

### CODE REVIEW

File: src/options.cpp [hunk 1]
```diff
@@ -48,6 +48,13 @@ void unc_add_option(const char *name, uncrustify_options id, argtype_e type,
                     const char *short_desc, const char *long_desc,
                     int min_val, int max_val)
 {
+#define OptionMaxLength 60
+   int lengthOfTheOption = strlen(name);
+   if (lengthOfTheOption > OptionMaxLength) {
+      fprintf(stderr, "FATAL: length of the option name (%s) is too big (%d)\n", name, lengthOfTheOption);
+      fprintf(stderr, "FATAL: the maximal length of an option name is %d characters\n", OptionMaxLength);
+      exit(EXIT_FAILURE);
```
Comment (mihaipopescu) [inline src/options.cpp]: I wish we could have this as a compile time assert.

File: src/options.cpp [hunk 2]
```diff
@@ -48,6 +48,13 @@ void unc_add_option(const char *name, uncrustify_options id, argtype_e type,
                     const char *short_desc, const char *long_desc,
                     int min_val, int max_val)
 {
+#define OptionMaxLength 60
```
Comment (mihaipopescu) [inline src/options.cpp]: I would have used an anonymous enums for constants

**Comment by mihaipopescu** [comment]
IIRC I think I've seen a similar code from Ben some while ago.

File: src/options.cpp (hunk 1 above)
Comment (gmaurel) [inline src/options.cpp]: How?

File: src/options.cpp (hunk 2 above)
Comment (gmaurel) [inline src/options.cpp]: I don't like anonymous enums!

**Comment by gmaurel** [comment]
I wait a little for you.

**Pull request merged**